docs/WinRT_API.md

```diff
@@ -1,14 +1,14 @@
 # Windows Machine Learning WinRT API
 
-New in the ONNX Runtime Nuget package is the ability to use the full [Windows.AI.MachineLearning API](https://docs.microsoft.com/en-us/windows/ai/windows-ml/api-reference).
+New in the ONNX Runtime Nuget package is the ability to use the full [WinML API](https://docs.microsoft.com/en-us/windows/ai/windows-ml/api-reference).
```

Comment:
What's the purpose of shortening this naming? Windowsi.AI.MachineLearning->WinML

Reply:
Windows.AI.MachineLearning refers specifically to the Inbox SDK Windows namespace, while WinML refers to the onnxruntime WinRT projection which may exist inbox in the WIndows namespace or the Microsoft namespace.